MaxText/input_pipeline/_grain_data_processing.py

```diff
@@ -44,22 +44,26 @@ def preprocessing_pipeline(
     grain_worker_count: int,
     dataloading_host_index,
     dataloading_host_count,
+    data_column,
     shuffle: bool = False,
     data_shuffle_seed=0,
+    tokenize=True,
     add_bos=True,
     add_eos=True,
     num_epochs=1,
     packing=True,
     shift=True,
-    drop_remainder=True,
+    drop_remainder=False,
```

Comment:
What does drop_remainder do? Could we include a comment/documentation somewhere?

Reply:
Sure, I'm still working on features related to drop_remainder. Will add comment/doc.